package.json

```diff
@@ -43,6 +44,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "mocha": "^10.0.0",
     "nodemon": "^2.0.19",
+    "object-assign": "^4.1.1",
```

Comment:
what's `object-assign` adding here?

Reply:
because a dependency for cors(), was throwing an error. According to github issues I needed to add it as devDependency, I will give it another go in a sec and feedback to you.

Reply:
It has been removed and thanks for flagging this one up. I think mine package-lock.json was a little off, so removing and `npm i` sorted this out